src/Components/NavBar/NavBar.css

```diff
@@ -0,0 +1,18 @@
+.NavBar {
+  margin: 2em;
+  display: flex;
+  position: fixed;
+  justify-content: space-around;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.NavBar a {
+  color: black;
+  text-decoration: none;
+}
```

Comment:
Something to think about: when a sighted visitor first encounters your app, it may not be obvious that your links are interactive. Here's a screenshot of what they look like in my browser:
![Screen Shot 2020-10-07 at 16 14 15](https://user-images.githubusercontent.com/13525251/95397545-5bacfb80-08b8-11eb-9656-299b5f16027f.png)

This looks like plain text, and that's a little misleading! When you redesign your links, think about ways to make them your own but still **look** like links. We web authors don't like the default blue-text-with-underline look too much, but it _is_ useful: the color contrast means something to us, and the underline helps those of us with color-deficient vision.

You have **tons** of creative freedom when designing your own links, and we can absolutely collaborate on designs in the future. This is just some food for thought.

Reply:
@mxmason Thanks for looking this over! We purposely tried not doing much styling because it was mentioned in the kickoff call to not focus much on it because we'll do it in the end. So you recommend we add some simple styles as we go to edit as needed later?

Reply:
Yes! Don't let styling slow you down right now; just do the minimum that helps you make sense of the UI as it grows. Keep this feedback in mind for whenever you're ready to style your links. :)